test/e2e/storage/persistent_volumes-local.go

```diff
@@ -161,23 +379,23 @@ func cleanupLocalVolume(config *localTestConfig, volume *localTestVolume) {
 	By("Cleaning up PVC and PV")
 	errs := framework.PVPVCCleanup(config.client, config.ns, volume.pv, volume.pvc)
 	if len(errs) > 0 {
-		framework.Logf("AfterEach: Failed to delete PV and/or PVC: %v", utilerrors.NewAggregate(errs))
+		framework.Failf("Failed to delete PV and/or PVC: %v", utilerrors.NewAggregate(errs))
 	}
 
 	By("Removing the test directory")
 	removeCmd := fmt.Sprintf("rm -r %s", volume.containerDir)
-	runLocalUtil(config, volume.node.Name, removeCmd)
+	runLocalUtil(config, volume.node.Name, removeCmd, containerBase)
 }
 
-func runLocalUtil(config *localTestConfig, nodeName, cmd string) {
+func runLocalUtil(config *localTestConfig, nodeName, cmd, basePath string) {
```

Comment:
This func seems a bit misleading to me. I expected you to create a single storage server pod (since that is the original purpose of this func AFAIK). Instead a busybox pod is created for each command to be executed. Have you considered creating a single cmd-exec pod for the lifespan of an `It` block?  Maybe something like this would work `CreateExecPodOrFail()`?